packages/account/src/Sections/Security/Passwords/deriv-email.tsx

```diff
@@ -1,68 +1,66 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { Formik } from 'formik';
-import { withRouter } from 'react-router';
-import { WS, toTitleCase } from '@deriv/shared';
-import { Localize, localize } from '@deriv/translations';
 import { Button, Text, Input } from '@deriv/components';
+import { useVerifyEmail } from '@deriv/hooks';
+import { toTitleCase } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
 import FormSubHeader from 'Components/form-sub-header';
 import SentEmailModal from 'Components/sent-email-modal';
 import UnlinkAccountModal from 'Components/unlink-account-modal';
-import { observer, useStore } from '@deriv/stores';
 
-const DerivEmail = observer(({ email, social_identity_provider, is_social_signup }) => {
-    const { common } = useStore();
-    const { is_from_derivgo } = common;
+/**
+ * Display the user's email address and a button to change it.
+ * @name DerivEmail
+ * @returns {React.ReactNode}
+ */
+const DerivEmail = observer(() => {
+    const {
+        common: { is_from_derivgo },
+        client: { social_identity_provider, is_social_signup, email },
+    } = useStore();
+    const { send } = useVerifyEmail('request_email');
     const [is_unlink_account_modal_open, setIsUnlinkAccountModalOpen] = React.useState(false);
     const [is_send_email_modal_open, setIsSendEmailModalOpen] = React.useState(false);
 
     const onClickChangeEmail = () => {
         if (is_social_signup) {
             setIsUnlinkAccountModalOpen(true);
         } else {
-            WS.verifyEmail(email, 'request_email');
+            send(email);
             setIsSendEmailModalOpen(true);
         }
     };
 
     const onClickSendEmail = () => {
-        WS.verifyEmail(email, 'request_email');
+        send(email);
         setIsUnlinkAccountModalOpen(false);
         setIsSendEmailModalOpen(true);
     };
 
-    const onClickResendEmail = () => {
-        WS.verifyEmail(email, 'request_email');
-    };
```

Comment:
Why?

Reply:
The function is not needed as it has been improved in refatoring